build/includes/sdk.mk

```diff
@@ -143,9 +148,11 @@ run-sdk-conformance-test: ensure-build-sdk-image
 
 # Run a conformance test for all SDKs supported
 run-sdk-conformance-tests:
+	$(MAKE) run-sdk-conformance-test SDK_FOLDER=restapi HTTP_PORT=9050
 	$(MAKE) run-sdk-conformance-test SDK_FOLDER=node
 	$(MAKE) run-sdk-conformance-test SDK_FOLDER=go   PORT=9001
 	$(MAKE) run-sdk-conformance-test SDK_FOLDER=rust
+	$(MAKE) run-sdk-command COMMAND=clean SDK_FOLDER=restapi
```

Comment:
The node sdk folder has a clean.sh script but it isn't called after running the tests. 

Should it be? Or this one removed? Either way, the pattern for testing the sdks should be consistent.

Reply:
I think the reason with Node js was that we gitignored the results of clean. Will check it out soon.

Reply:
Yes, this step is needed, because it also delete a copy of `test/sdk/restapi/http-api-test.go.nolint` which becomes `http-api-test.go` and would result in errors on `make lint` - about 50 linting errors would be displayed, all from swagger generated code.